WP2RDF/src/main/java/org/wikipathways/wp2rdf/GpmlConverter.java

```diff
@@ -147,6 +148,22 @@ public static void getUnifiedIdentifiers(Model model, IDMapper  mapper, Xref idX
 				outputBridgeDbMapping(model, mapper, idXref, internalWPDataNodeResource,
 					"Lm", "http://identifiers.org/lipidmaps/", Wp.bdbLipidMaps
 				); 
+		// InChIKey
+		if (mapper != null) {
```

Comment:
Please integrate this into the `outputBridgeDbMapping` method.

Or is there a reason to do it differently?

Reply:
You mean like here for lipidmaps? https://github.com/wikipathways/GPML2RDF/commit/80fe0ed31146cca38a7f8ea79406d42bc93d0f5b . I'm not sure why you decided to write it like you did (the commit on inChIKeys is yours ;)....) . Doesn't that also have something to do with wanting neutral inchikeys?

Reply:
hahaha... lol

Reply:
Thought you might have forgotten to integrate this chance, since I didn't see it back in the RDF data I'm currently working with....